lib/utils.js

```diff
@@ -2,22 +2,48 @@
 
 const { request } = require('undici')
 
-async function refreshAccessToken ({ refreshEndpoint, refreshToken, clientId }) {
-  const { statusCode, body } = await request(`${refreshEndpoint}/token`, {
-    method: 'POST',
-    headers: {
-      'Content-Type': 'application/json'
-    },
-    body: JSON.stringify({
+async function refreshAccessToken ({ idpTokenUrl, refreshToken, clientId, clientSecret, contentType }) {
```

Comment:
Keeping the application/json option in there is unnecessary, given that every Authorization Server is required to implement `application/x-www-form-urlencoded`. `application/json` is not coming from any normative requirement.

Reply:
I know. We have some old code we rely on that I don't want to break. I'll add a TODO to remove this